specification/securitycenter/resource-manager/Microsoft.Security/preview/2017-08-01-preview/security.json

```diff
@@ -0,0 +1,1370 @@
+        "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+        "version": "2017-08-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": ["https"],
+    "consumes": ["application/json"],
+    "produces": ["application/json"],
+    "security": [{
+        "azure_auth": [
+            "user_impersonation"
+        ]
+    }],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/providers/Microsoft.Security/operations": {
+            "get": {
+                "tags": ["Operations"],
+                "description": "Exposes all available operations for discovery purposes.",
+                "operationId": "Operations_List",
+                "parameters": [{
+                    "$ref": "#/parameters/ApiVersion"
+                }],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/OperationList"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Security/pricings": {
+            "get": {
+                "tags": ["Pricings"],
+                "description": "Security pricing configurations in the subscription",
+                "operationId": "PricingsSubscription_List",
+                "parameters": [{
+                    "$ref": "#/parameters/ApiVersion"
+                }, {
+                    "$ref": "#/parameters/SubscriptionId"
+                }],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/PricingList"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/pricings": {

```

Comment:
I recall having the discussion with folks on your side around why cant this be exposed through the SKU. The scenario is slightly different here in that the pricing doesnt apply to resources under a provider. And it was mentioned that MarkRuss and folks are aware and approved this model :). 
Just wanted to take a note so that everyone is on the same page. Also in future if we need to reference this, we know why this is designed in a different way.